### DISCUSSION TITLE
Installation support

### DISCUSSION THREAD

**Discussion by 99003662** [google/googletest#3730] [Community Help]
Hi team,
              Am working in VS code for one of my project. So we decided to use gtest for unit test of our code which is in C++. We went through all the supported doc and still we are not able to setup the configuration completely. Is there any reference to follow.

**Comment by PRINCE-67** [comment]
hello, I think this link will help you.

https://www.youtube.com/watch?v=n1OL800IcXI

**Comment by rishabhdeepsingh** [comment]
@opPRINCE69 Visual sudio != VS code
